### PR TITLE
[api] SourceController#project_command_release uses ActiveJob

### PR DESCRIPTION
**Pull request by evanrolfe**
Part of this card: https://trello.com/c/Tx9d3bUb/467-5-p4-event-system-change-delayed-jobs-to-be-created-with-activejobs

### CODE REVIEW

File: src/api/app/controllers/source_controller.rb [hunk 1]
```diff
@@ -1006,7 +1006,7 @@ def project_command_release
       render_ok
     else
       # inject as job
-      @project.delay.do_project_release(params)
+      ProjectDoProjectReleaseJob.perform_later(@project.id, params)
```
Comment (hennevogel) [inline src/api/app/controllers/source_controller.rb]: `queue_as :quick`! :-)

File: src/api/app/jobs/project_do_project_release_job.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class ProjectDoProjectReleaseJob < ApplicationJob
+  queue_as :quick
+
+  def perform(project_id, params)
+    ProjectDoProjectReleaseJob.perform_later(
```
Comment (hennevogel) [inline src/api/app/jobs/project_do_project_release_job.rb]: Erm a job that calls itself?

Comment (evanrolfe) [inline src/api/app/jobs/project_do_project_release_job.rb]: Arghh I copied bjoern's [fix](https://github.com/evanrolfe/open-build-service/pull/6/files) into the wrong place. I will sort this out tomorrow morning, it seems my brain is getting tired...

**Comment by evanrolfe** [comment]
@hennevogel please re-review

**Pull request merged**